.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+**Release note**:
+<!--  Steps to write your release note:
+1. Use the release-note-* labels to set the release note state (if you have access)
```

Comment:
Thx! I wanted to ask that during the demo, what does release-note-none mean/is it a good thing ? (my guess is it means skip the PR in release notes ? can you confirm ? so should be the exception right ?)

also what are the other release-note-* we envision (beside none and needed)